web/template/layout/mlpa-case-summary.gohtml

```diff
@@ -0,0 +1,60 @@
+{{ define "mlpa-case-summary" }}
+    <div class="app-!-caseworker-summary">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-full">
+                <div class="govuk-!-display-block">
+                    <h3 class="govuk-heading-s govuk-!-display-inline govuk-!-padding-right-3">Case summary</h3>
+                    <button type="button" class="govuk-accordion__show-all govuk-!-margin-0" data-module="app-case-summary-toggle">
+                        <span class="govuk-accordion-nav__chevron govuk-accordion-nav__chevron--up case-summary-toggle-icon"></span>
+                        <span class="case-summary-toggle-text">Hide</span>
```

Comment:
This won't show the words "case summary" on screen, but means that a screen reader will announce the button content as "Hide case summary". Otherwise if a user comes across this button they won't know what it is they're hiding.
```suggestion
                        <span class="case-summary-toggle-text">Hide</span>
                        <span class="govuk-visually-hidden">case summary</span>
```